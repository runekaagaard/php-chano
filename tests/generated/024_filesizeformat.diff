001- 1.0 MB
001+ 1.0 MiB