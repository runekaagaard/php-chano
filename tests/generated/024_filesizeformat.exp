1.0 MB