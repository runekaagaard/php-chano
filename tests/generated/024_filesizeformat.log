
---- EXPECTED OUTPUT
1.0 MB
---- ACTUAL OUTPUT
1.0 MiB
---- FAILED
