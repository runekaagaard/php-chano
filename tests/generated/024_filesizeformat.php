<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 1048576)));
foreach ($items as $i) echo $i->input->filesizeformat();
