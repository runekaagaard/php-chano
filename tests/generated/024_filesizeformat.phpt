--TEST--
A generated testfile for the "filesizeformat" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 1048576)));
foreach ($items as $i) echo $i->input->filesizeformat();
--EXPECT--
1.0 MB