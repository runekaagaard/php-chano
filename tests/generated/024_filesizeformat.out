1.0 MiB